app/src/main/java/org/thoughtcrime/securesms/ui/theme/ThemeColorSet.kt

```diff
@@ -5,6 +5,8 @@ package org.thoughtcrime.securesms.ui.theme
  * light theme, and [dark] representing the [ThemeColors] to use when the system is in a dark theme.
  */
 data class ThemeColorSet(
-    val light: ThemeColors,
-    val dark: ThemeColors
-)
\ No newline at end of file
+    val colorsWhenSystemInLight: ThemeColors,
```

Comment:
Those names are misleading since we can use those fields regardless of the system settings.
I think `dark` and `light` are simple enough and match what they are.

Reply:
it was true, how I changed the code, but anyway, I deleted this class, am using a lambda now.